lib/pod/command/plugins/installed.rb

```diff
@@ -41,25 +42,46 @@ def print_compact_list(plugins)
           max_length = plugins.map { |p| p.name.length }.max
           plugins.each do |plugin|
             name_just = plugin.name.ljust(max_length)
-            UI.puts_indented " - #{name_just} : #{plugin.version}"
+            hooks = registered_hooks(plugin)
+            hooks_list = hooks.empty? ? '' : " (#{hooks.join(' & ')} hook)"
+            UI.puts_indented " - #{name_just} : #{plugin.version}#{hooks_list}"
           end
         end
 
-        # Print the given plugins as a verbose list,
-        #    with name, version, homepage and summary
-        #    for each plugin.
+        # Print the given plugins as a verbose list, with name, version,
+        #   homepage and summary for each plugin.
         #
         # @param [Array<Gem::Specification>] plugins
         #        The list of plugins to print
         #
         def print_verbose_list(plugins)
           plugins.each do |plugin|
+            hooks = registered_hooks(plugin)
+
             UI.title(plugin.name)
-            UI.labeled('Version', plugin.version)
-            UI.labeled('Homepage', plugin.homepage) if plugin.homepage
+            UI.labeled('Version', plugin.version.to_s)
+            UI.labeled('Hooks', hooks) unless hooks.empty?
+            unless plugin.homepage.empty?
+              UI.labeled('Homepage', plugin.homepage)
+            end
             UI.labeled('Summary', plugin.summary)
           end
         end
+
+        # List of registered hook(s) (if any) for the given plugin
```

Comment:
Scratch that, just confused myself because the documentation is wrong in CocoaPods itself.